### ISSUE TITLE
Locked out of forum.

### ISSUE THREAD

**Issue by BallscrewBob** [arduino/forum-issues#237]
Sign in option is doing nothing when clicked on.
However I am fully logged on in the Create Editor.

There are no other messages to aid in diagnostics
Below was pulled from the chrome console on trying to use the sign in option

`jquery.min.js:3 Unsafe attempt to load URL https://cdn.arduino.cc/header-footer/6xm6gycik9/create-nav-images/icons.svg from frame with URL https://forum.arduino.cc/?code=-nUQDR8Lubj79_Z4__PuQwFP_FT8UaGanD0TBYbZF2w.…ile%3Acore%20profile%3Apublic%20profile%3Acontact%20offline&state=90785687. Domains, protocols and ports must match.

(anonymous) @ jquery.min.js:3
n.access @ jquery.min.js:3
html @ jquery.min.js:3
(anonymous) @ jquery.min.js:4
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
b @ jquery.min.js:4
404.html:1 Uncaught SyntaxError: Unexpected token <
cron.php:1 GET https://forum.arduino.cc/cron.php?ts=1494468450 500 (Internal Server Error)
?code=-nUQDR8Lubj79_Z4__PuQwFP_FT8UaGanD0TBYbZF2w.pJMi1nHl4LA1mN5cbL14WjFxXsnlXRd8Kt7zqOATiYQ&scope…:1 XMLHttpRequest cannot load https://store.arduino.cc/usa/cart/api/count. Redirect from 'https://store.arduino.cc/usa/cart/api/count' to 'https://auth.arduino.cc/login?response_type=code&client_id=retail&redirect_…ofile%3Acontact+profile%3Aaddress+profile%3Aprivate&state=RoxKBuVEFGDwvCQ1' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://forum.arduino.cc' is therefore not allowed access.
header.js:55 Object {readyState: 0, getResponseHeader: function, getAllResponseHeaders: function, setRequestHeader: function, overrideMimeType: function…}
error @ header.js:55
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
b @ jquery.min.js:4
?code=-nUQDR8Lubj79_Z4__PuQwFP_FT8UaGanD0TBYbZF2w.pJMi1nHl4LA1mN5cbL14WjFxXsnlXRd8Kt7zqOATiYQ&scope…:1 XMLHttpRequest cannot load https://hydra.arduino.cc/oauth2/auth?client_id=forum&redirect_uri=https%3A%…ofile%3Acore+profile%3Apublic+profile%3Acontact+offline&response_type=code. Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access.
header.js:97 Object {readyState: 0, getResponseHeader: function, getAllResponseHeaders: function, setRequestHeader: function, overrideMimeType: function…}
error @ header.js:97
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
b @ jquery.min.js:4
?code=-nUQDR8Lubj79_Z4__PuQwFP_FT8UaGanD0TBYbZF2w.pJMi1nHl4LA1mN5cbL14WjFxXsnlXRd8Kt7zqOATiYQ&scope…:1 XMLHttpRequest cannot load https://store.arduino.cc/usa/cart/api/minicart. Redirect from 'https://store.arduino.cc/usa/cart/api/minicart' to 'https://auth.arduino.cc/login?response_type=code&client_id=retail&redirect_…ofile%3Acontact+profile%3Aaddress+profile%3Aprivate&state=ZpMEzReLss5Uy0rH' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://forum.arduino.cc' is therefore not allowed access.
header.js:71 Can't fetch cart data
error @ header.js:71
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
b @ jquery.min.js:4
header.js:72 Object {readyState: 0, getResponseHeader: function, getAllResponseHeaders: function, setRequestHeader: function, overrideMimeType: function…}
`

**Comment by BallscrewBob** [comment]
EDIT to above..By going to first the EDITOR and then through to Project HUB and then back from project hub to the forum I am logged in.
The issue only seems to affect direct access.

**Comment by BallscrewBob** [comment]
![forum down no sign in available also errors at bottom of main page](https://cloud.githubusercontent.com/assets/8212730/25929942/3cb496e2-35d1-11e7-840f-5922ef1f3d61.jpg)

**Comment by sterretje** [comment]
I can not confirm the details above but sign-in does not take me anywhere. No menu to view e.g. updated topics.

Tried incorrect password and that at least tells me that I'm not who I say I am ;) Tried correct password and that is accepted. Clicking on one of the categories after that goes though the motions and your back on the main forum page.

Android 6 with Chrome 58.0.3029.83, Windows 7 with Firefox 53

Browser caches and cookies cleared, it does not make a difference.

**Comment by sterretje** [comment]
To add to this

Did a search in the arduino forum search box (for my username) and got to the usual google page
Click a link
Takes me to the forum main page, not to the thread / post

Please put a message on the forum main page that the site is down for maintenance or whatever, it's now totally useless.

When the problems are sorted, re-instate the main page.

**Comment by sterretje** [comment]
Yes, I'm in !!

The solution **might** be

Don't login (clear cookies etc if you did)
Follow a link to the forum (e.g from a normal google search)
Login

**Comment by Serjlee** [comment]
Clearly, those kind of issues happen while over there is the middle of night. 
Is someone still locked out?

@sterretje can you reproduce the redirect issue? I'm unable to. (Open a new issue, if you can reproduce it)

**Comment by BallscrewBob** [comment]
I am back in this morning with no changes.

I had done the full clean up, cache cookies etc etc etc. last night as that is now a first course of action with any forum issues.

**Comment by sterretje** [comment]
@Serjlee 

Can't reproduce at this stage, it was related to the login problems described above.

I assume that you can mark this as solved in some way ;)

**Comment by BallscrewBob** [comment]
Yes mark as "closed" will re-comment if it ever comes up again.
